packages/wallet-sdk/src/sign/SignerConfigurator.ts

```diff
@@ -26,11 +33,11 @@ export class SignerConfigurator {
   private popupCommunicator: PopUpCommunicator;
   private updateListener: SignRequestHandlerListener;
 
-  private signerTypeStorage = new ScopedLocalStorage('CBWSDK', 'SignerConfigurator');
-  private signerTypeSelectionResolver: ((signerType: string) => void) | undefined;
+  private injectedSigner: Signer | undefined;
+  private initiatedWLSigner: WLSigner | undefined;
```

Comment:
seems we need to distinguish
initiate v.s. select a signer

we expect only one signer to be selected at a time.
but at the same time, we might be initiating more than one of them, e.g. initiating WL for qrcode.

Reply:
updated naming in this area!